src/jsonformat.erl

```diff
@@ -83,7 +93,14 @@ jsonify(Any)                   -> unicode:characters_to_binary(io_lib:format("~w
 
 a2b(A) -> atom_to_binary(A, utf8).
 
-new_line(Config)        -> maps:get(new_line, Config, ?NEW_LINE).
+format_data(Data, #{format_funs := Callbacks}) ->
+  maps:fold(fun(K, Fun, Acc) ->
+              maps:update_with(K, Fun, Acc)
```

Comment:
If the message does not contain the key for a callback, this code will crash. This is not really an issue if you're formatting a meta key (e.g. `time`) because these will always exist. However, if one would like to format a key from the user defined part of the log then we cannot assume that the key will always exists. 

I suggest that we add a check with `maps:is_key/2` before we call `update_with/3`, or we can wrap `update_with/3` in a try/catch statement.

Reply:
I think I addressed that in the meantime

Reply:
Yes you did, I always forget about the `is_map_key/2` guard expression...